docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -141,8 +128,7 @@ hcp create cluster openstack \
 --openstack-credentials-file $CLOUDS_YAML \
 --openstack-external-network-id $EXTERNAL_ID \
 --openstack-node-image-name $IMAGE_NAME \
---openstack-node-flavor $FLAVOR \
```

Comment:
is this needed to be removed?

Reply:
It hasn't been removed. Only `--openstack-ingress-floating-ip` has been removed.